apps/api/src/api/application-api.ts

```diff
@@ -108,60 +108,60 @@ export const getApplicationById = async ({ applicationId }: { applicationId: num
 };
 
 export const approveApplication = async ({
-  applicationId,
-  approverId,
+	applicationId,
+	approverId,
 }: ApproveApplication): Promise<{
-  success: boolean;
-  message?: string;
-  errors?: string | Error;
-  data?: any;
+	success: boolean;
+	message?: string;
+	errors?: string | Error;
+	data?: any;
 }> => {
-  try {
-    // Fetch application
-	const database = getDbInstance();
-	const service: ApplicationService = applicationService(database);
-	const result = await service.getApplicationById({ id:applicationId });
-
-	if (!result.success) {
-		return {
-			success: false,
-			message: 'Application not found.',
-			errors: 'ApplicationNotFound',
-		  };
-	}
-
-	const { state } = result.data;
-	if(state ===  ApplicationStates.APPROVED){
-		return {
-			success: false,
-			message: 'application is already approved.',
-			errors: 'ApprovalConflict',
-		  };
-	}
+	try {
+		// Fetch application
+		const database = getDbInstance();
+		const service: ApplicationService = applicationService(database);
+		const result = await service.getApplicationById({ id: applicationId });
+
+		if (!result.success) {
+			return {
+				success: false,
+				message: 'Application not found.',
+				errors: 'ApplicationNotFound',
+			};
```

Comment:
This can just be `(!result.success) { return result }`
The success, errors and message will already be populated by getApplicationById